servicetalk-http-api/src/main/java/io/servicetalk/http/api/HttpExecutionStrategies.java

```diff
@@ -246,8 +250,10 @@ public Builder offloadWithThreadAffinity() {
          *
          * @param mergeStrategy {@link MergeStrategy} to use.
          * @return {@code this}.
+         * @deprecated Will be removed
          */
         // Intentionally package-private, API is not required to be public for the lack of use cases.
+        @Deprecated
```

Comment:
This is pkg-private API. Ok to deprecate it, but won't matter for users either way